data/gisaidCounts.ipynb

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
If you say the answer is yes, that's fine, but do we want this file committed? It looks like this is some Jupyter notebook stuff?

Reply:
Yes - this is the script I use to wrangle the raw gisaid metadata dumps from cz gen epi --> the counts file we use (I almost lost it recently). It can totally live as a .py if needed, but I like running these things interactively anyways.